.github/workflows/est-ds-realm-test.yml

```diff
@@ -392,6 +392,32 @@ jobs:
       - name: Enroll certificate with user/password but using different subject and check disabled
         run: |
           docker exec pki mkdir new_certs3
+          docker exec -e EST_OPENSSL_CACERT=cacert.pem pki estclient -e -s pki.example.com -p 8443 \
+              --common-name test-new.example.com -o new_certs3 -u est-test-user -h Secret.123 | tee output || true
+
+          # Request should fails with bad request error (status code 404)
+          STATUS=$(cat output  | sed -nE 's/.* HTTP response from EST server was (.*)$/\1/p')
+          [ "$STATUS" == "BAD REQUEST" ]
```

Comment:
Should we check for a specific return code (e.g. 401 or 403) or just any 4xx code (bad request) would be accepted? Not sure if 404 (not found) is the proper code for this scenario. Same thing with the next test below.

Reply:
The current return code here is 400.  The CA endpoint is identified and process the request but the output is not good. In the 4xx error options I think this is acceptable in this case, the other are more specific but for different cases.

The status is printed by `estclient` and it writes the error code number for some errors while write the error name for other so we cannot get the number in this case.

Reply:
OK, it's just the comment says `status code 404`, that probably should be updated.